pages/button/button.css

```diff
@@ -31,3 +42,30 @@
   background-color: var(--danger-color);
   color: var(--white-color);
 }
+/* icon buttons  */
+.btn i {
+  padding: 5px;
+  /* color: var(--white-color); */
+}
+/* floating button example used in HTML */
```

Comment:
At most places there is one empty line after each code block. However, it is  missed at some places like line 44, 48, 68 and some more. Just my thought.